### DISCUSSION TITLE
TuringCourses/short_term/ctf/

### DISCUSSION THREAD

**Discussion by giscus** [ZJU-Turing/TuringCourses#47] [General]
# TuringCourses/short_term/ctf/

图灵班学习指南

https://zju-turing.github.io/TuringCourses/short_term/ctf/

**Comment by WhiteSaikou** [comment]
神中神的课程，给分也很好

**Comment by NeilAlfred** [comment]
2023 年暑假刚上完，回来评论，神课！建议有代码基础的同学都可以去尝试！

**Comment by Thorin215** [comment]
好课bd
痛并快乐着
f0rm2l1n我的神!

**Comment by aosdjao** [comment]
bd，时间还给延了好久，只要上课认真听，课后作业积极做，不赶ddl还是很好满绩的，要是实在不太感兴趣，给的保底分也很高了。
